website/docs/d/google_organization.html.markdown

```diff
@@ -22,13 +22,15 @@ resource "google_folder" "sales" {
 }
 ```
 
+Note that the identity whose credentials are used for authentication needs to have `roles/resourcemanager.organizationViewer` role to be able to read that info.
+
 ## Argument Reference
 
 The arguments of this data source act as filters for querying the available Organizations.
 The given filters must match exactly one Organizations whose data will be exported as attributes.
 The following arguments are supported:
 
-* `organization` (Optional) - The name of the Organization in the form `{organization_id}` or `organizations/{organization_id}`.
+* `organization` (Optional) - The numeric ID of the Organization in the form `{organization_id}` or `organizations/{organization_id}`.
```

Comment:
```suggestion
* `organization` (Optional) - The name (numeric ID) of the Organization in the form `{organization_id}` or `organizations/{organization_id}`.
```
`name` is the actual term for the field here, as it is defined here: https://cloud.google.com/resource-manager/reference/rest/v1/organizations/get

So I'm hesitant to remove it completely. I understand the want for clarification, do you believe this suggestion will suffice?

Reply:
Sure that makes sense. Initially I thought it means I can use either organization name or ID, that's why I thought it'd be good to clarify. Your suggestion works for me, thanks!